src/console/Microsoft.Health.Fhir.Ingest.Console.Common/Extensions/CredentialProviderExtensions.cs

```diff
@@ -1,8 +1,7 @@
-﻿// --------------------------------------------------------------------------
-// <copyright file="CredentialProviderExtensions.cs" company="Microsoft Corporation">
+﻿// -------------------------------------------------------------------------------------------------
 // Copyright (c) Microsoft Corporation. All rights reserved.
-// </copyright>
-// --------------------------------------------------------------------------
+// Licensed under the MIT License (MIT). See LICENSE in the repo root for license information.
```

Comment:
Quick look it does appear we are using the MIT header in other places in the code, these is just to correct the few places where we weren't?

Reply:
I got these compilation errors to fix licensing (likely because either the compiler toolset or stylecop analyzer package got applied to these projects that didn't previously have it) and it matches the other files in this repo now.